radio/src/storage/conversions/conversions_218_219.cpp

```diff
@@ -54,7 +54,7 @@ int convertSource_218_to_219(int source)
 #endif
 
 #if defined(PCBHORUS)
-  if (source >= MIXSRC_GMBL)
+  if (source >= MIXSRC_SI)
```

Comment:
Would you add a non regression test for X10 translations? As we did for X7 / X9D+ / X9E

Reply:
OK the non regression test is already there, perhaps even done by me, I cannot remember